src/winter_onboarding_2021/fleet_management_service/views/layout.clj

```diff
@@ -2,12 +2,14 @@
   (:use [hiccup.page :only (html5 include-css include-js)]
         [winter-onboarding-2021.fleet-management-service.views.content :as content]))
 
-(defn application [title & content]
+(defn application [request title & content]
   (html5 [:head
           [:title title]
           [:link {:rel "icon" :href "public/favicon.ico" :type "image/x-icon"}]
-          (include-css "public/css/bootstrap.min.css")
-          (include-js "public/js/jquery-3.6.0.min.js")
-          (include-js "public/js/bootstrap.bundle.min.js")
+          (include-css "/public/css/bootstrap.min.css")
+          (include-js "/public/js/jquery-3.6.0.min.js")
+          (include-js "/public/js/bootstrap.bundle.min.js")
           [:body
-           [:div {:class "container"} content]]]))
+           [:div {:class "container"}
+            [:div {:class (get-in request [:flash :style-class])} (get-in request [:flash :message])]
```

Comment:
Hey @shhivam , do we require flash for every application layout? I think making another application layout for this use case might make more sense?

Reply:
To not repeat same Alert banner HTML in every component, we thought it would be best if this HTML was at layout level

Reply:
Ok. makes sense.